app/marketing/management/commands/assemble_leaderboards.py

```diff
@@ -290,6 +290,8 @@ def sum_tip_helper(t, time, index_term, val_usd):
 
 def sum_kudos(kt):
     val_usd = kt.value_in_usdt_now
+    if not kt.kudos_token_cloned_from:
+        return
```

Comment:
hmm.. what does this logic do here?

Reply:
there is a kudos transfer that has a null `kudos_token_cloned_from` and this logic prevents an exception here.

Reply:
ah nice thanks for the info